[Bug]: Can not build namui-drawer in github action

### Expected Behavior

success

### Actual Behavior

```text
  = note: /usr/bin/ld: cannot find -lfontconfig: No such file or directory
          /usr/bin/ld: cannot find -lfreetype: No such file or directory
          collect2: error: ld returned 1 exit status
```

### Steps to Reproduce

run ci

### Other Info

Runner's os is changed from ubuntu 22 to 24.
- [before](https://github.com/NamseEnt/namseent/actions/runs/12227497558/job/34104312160)
- [current](https://github.com/NamseEnt/namseent/actions/runs/12512479530/job/34905851168)

--- a/.github/workflows/namui-linux-host-x86_64-unknown-linux-gnu.yml
+++ b/.github/workflows/namui-linux-host-x86_64-unknown-linux-gnu.yml
@@ -42,7 +42,7 @@ jobs:
       - name: Install dependencies
         run: |
           sudo apt-get update
-          sudo apt-get install -y libasound2-dev
+          sudo apt-get install -y libasound2-dev libfontconfig-dev
 
       - name: Set up cargo
         run: |

